mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -93,6 +115,7 @@ def get_object(self):
         """
         try:
             obj = self.session.get(self.get_object_endpoint_path()).json()
+            obj['needs_attention'] = parse_datetime(obj['credit']['started_at']) < self.need_attention_date
             return convert_date_fields(obj, include_nested=True)
```

Comment:
the `convert_date_fields` parses the `started_at` value, so you could just add the `needs_attention` flag after